docs/source/commands/dev.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: The Rover dev command
+description: Query a locally federated GraphQL API
+---
+
+You can use Rover to start a local router that can query across one or more running GraphQL APIs (i.e., subgraphs)
+through one endpoint (i.e., supergraph). As you add, edit, and remove subgraphs, Rover automatically
+composes their separate schemas into a unified supergraph schema that the router reloads and serves. 
+
+Think plug-n-play USB devices but with your GraphQL APIs!
+
+A **subgraph** is a graph that contributes to the composition of a federated **supergraph**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+> ⚠️ Do not run this command in production! ⚠️ This command is only intended for local development.
+
+## Creating a federated session
+
+### Starting the session
+
+Before starting `rover dev`, you need to start your GraphQL API (subgraph). Note the URL it is running on, and the location of the schema on the filesystem if it exists. If you don't have a local schema file, introspection will need to be enabled on the subgraph.
+
+An example `rover dev` command for running a local subgraph might look something like this:
+
+`rover dev --name products --schema ./products.graphql --url http://localhost:4000`
+
+The above command starts the `rover dev` session, watching `./products.graphql` for changes to the schema, and starts up a local Apollo Router at the default endpoint (i.e., `http://localhost:3000`). This session also listens for other `rover dev` processes that include different subgraph schemas.
+
+### Stopping the session
+
+If you stop the first `rover dev` session (by pressing ctrl+c), it will shut down the router and any attached `rover dev` sessions will be detached and shut down.
+
+### Attaching additional subgraphs to a session
+
+To add a new subgraph to a session, run `rover dev` in a new terminal with the necessary arguments for your GraphQL API (i.e., subgraph). If a session is already running, Rover automatically attaches subsequent `rover dev` processes to that existing session.
+
+When you add a new subgraph to a session, Rover handles recomposing the supergraph schema and printing the build result in the outputs of both the main session and the attached session.
```

Comment:
Changing these to process(along the lines of each `rover dev` process is part of the unified session)
```suggestion
When you add a new subgraph to a session, Rover handles recomposing the supergraph schema and printing the build result in the outputs of both the primary process and the attached process.
```